release_notes/ocp-4-0-release-notes.adoc

```diff
@@ -0,0 +1,778 @@
+[id='ocp-4-1-release-notes']
+= {product-title} 4.1 release notes
+include::modules/common-attributes.adoc[]
+:context: release-notes
+
+toc::[]
+
+== Overview
+
+Red Hat {product-title} provides developers and IT organizations with a hybrid
+cloud application platform for deploying both new and existing applications on
+secure, scalable resources with minimal configuration and management overhead.
+{product-title} supports a wide selection of programming languages and
+frameworks, such as Java, JavaScript, Python, Ruby, and PHP.
+
+Built on Red Hat Enterprise Linux and Kubernetes, {product-title}
+provides a more secure and scalable multi-tenant operating system for today’s
+enterprise-class applications, while delivering integrated application runtimes
+and libraries. {product-title} enables organizations to meet security, privacy,
+compliance, and governance requirements.
+
+[[ocp-4-1-about-this-release]]
+== About this release
+
+Red Hat {product-title} 4.1
+(link:https://access.redhat.com/errata/RHBA-2019:XXXX[RHBA-2019:XXXX]) is now
+available. This release is based on
+link:https://github.com/openshift/origin/releases/tag/v4.0.0-alpha.0[OKD 4.1],
+and it uses Kubernetes 1.13. New features, changes, and known issues
+that pertain to {product-title} 4.1 are included in this topic.
+
+{product-title} 4.1 is supported on Red Hat Enterprise Linux 8 and later.
+
+[[ocp-41-deprecated-features]]
+=== Deprecated features
+
+Large changes to the underlying architecture and installation process are
+applied in {product-title} 4.1, and many features from {product-title} 3.x are
+now deprecated.
+
+.Features Deprecated in Version 4.1
```

Comment:
imagepolicyadmission plugin configurability is...maybe not deprecated, but not present in 4.1 (configuration options may be reintroduced in the future).  The plugin itself runs, but only w/ the default configuration.  Configuring it requires using the unsupported overrides mechanism.

/cc @adambkaplan